.github/workflows/develop-ci.yml

```
@@ -106,3 +106,45 @@ jobs:
+    runs-on: ubuntu-latest
+    needs: [set-environment]
+    permissions:
+      id-token: write
+      contents: read
+    environment: ${{ needs.set-environment.outputs.environment }}
+    steps:
+      - name: checkout the code
+        uses: actions/checkout@v4
+      - name: AWS 설정
+        id: credentials
+        uses: aws-actions/configure-aws-credentials@v2
```

actionlint (1.7.7)

the runner of "aws-actions/configure-aws-credentials@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)